.cirrus.yml

```diff
@@ -210,8 +210,8 @@ build_each_commit_task:
 
     setup_environment_script: '$SCRIPT_BASE/setup_environment.sh |& ${TIMESTAMP}'
     build_each_commit_script:
-        - 'source $SCRIPT_BASE/lib.sh'
-        - 'git fetch --depth $CIRRUS_CLONE_DEPTH origin $CIRRUS_BASE_BRANCH |& ${TIMESTAMP}'
+        - 'source $SCRIPT_BASE/lib.sh &> /dev/null'
```

Comment:
??

Reply:
Cirrus brings all these YAML strings into a generated script with 'set -x', so sourcing that results splatting the entire file into the logs :scream: 

I need to look at the logs, but I was hoping that would make that not happen?

I may just need to dump this section into an actual script and call it instead.

Reply:
Added comment.